src/main/java/com/google/cloud/tools/app/impl/cloudsdk/internal/sdk/CloudSdk.java

```diff
@@ -97,24 +97,24 @@ private void outputCommand(List<String> command) {
     log.info("submitting command: " + joiner.join(command));
   }
 
-  private Path getSdkPath() {
+  private File getSdkPath() {
     return sdkPath;
   }
 
-  private Path getGCloudPath() {
-    return sdkPath.resolve(GCLOUD);
+  private File getGCloudPath() {
```

Comment:
Yes, but how? Also, I want to address the questions of whether we need all of these get...Path methods in a separate PR.